docs/leaderboard.rst

```diff
@@ -0,0 +1,120 @@
+Leaderboard
+=====================
+
+
+A Bebo component to quickly add a leaderboard into your HTML5 game. **NOTE: Leaderboard is scoped to group and app, so the leaderboard from one game won't interfere with another, and won't interfere with the same game on another group.***
```

Comment:
use proper `NOTE` reStructuredText notation here. see http://www.sphinx-doc.org/en/stable/rest.html for documentation and our own quick_start.rst and getting_started.rst for examples